kale/utils/logger.py

```diff
@@ -3,17 +3,17 @@
 import datetime
 import logging
 import os
-import shlex
-import subprocess
+import uuid
 
 
-def git_hash():
-    """Gets a hash for different runs to have unique logfile names."""
-    cmd = 'git log -n 1 --pretty="%h"'
-    ret = subprocess.check_output(shlex.split(cmd)).strip()
-    if isinstance(ret, bytes):
-        ret = ret.decode()
-    return ret
+def out_file_core():
+    """Creates an output file name concatenating a formatted date and uuid, but without an extension.
+
+    Returns:
+        string: A string to be used in a file name.
+    """
+    date = str(datetime.datetime.now().strftime("%Y%d%m_%H%M%S"))
+    return f"log-{date}-{str(uuid.uuid4())}"
```

Comment:
Or more concisely: `f"log-{datetime.datetime.now():%Y%d%m_%H%M%S}-{uuid.uuid4()}"`

Reply:
Thanks!

Reply:
I've kept it verbose for now.